app/templates/components/travis-status.hbs

```diff
@@ -1,6 +1,6 @@
-<h3 class="footer-title">Travis CI Status</h3>
-<ul>
-  <li><div class="status-circle {{appLoading.fetchTravisStatus.last.value}}">Status:</div>
-      <a href="http://www.traviscistatus.com/">Travis CI Status</a>
-  </li>
-</ul>
+<div class="status-circle">
+  &#9679;
```

Comment:
Hey @backspace, what do you think about this change? It draws black dot as a symbol instead of styled div. This makes the dot size to depend on `font-size` and scale along with the label. Are there any downsides or possible regressions that comes to your mind? Thanks!

Reply:
I can’t think of any problems with that approach, seems like a good solution to me. Maybe while you’re working on this you could add a tooltip or just `title` that shows the word for the status, if available? That would make it work better for people with trouble distinguishing colours. Better than that would be an icon, but that would need more input from others.

Reply:
@backspace thanks for response and for idea! I'll add the status description to tooltip 👍

Reply:
@backspace it's live in the PR deployment 🚀

Reply:
Nice, it looks great to me, thank you!